src/palace/manager/feed/admin.py

```diff
@@ -22,13 +26,16 @@ def suppressed(
 
         q = (
             _db.query(LicensePool)
+            .join(Work)
+            .join(Edition)
             .filter(
                 and_(
-                    LicensePool.suppressed == True,
-                    LicensePool.superceded == False,
+                    LicensePool.suppressed == false(),
+                    LicensePool.superceded == false(),
```

Comment:
@tdilauro is there a reason to use `false()` vs `False` here? I've not seen the `false()` function used in a comparison like this before.

Reply:
It's just a way to avoid PEP 8 complaints about comparison (`==` vs. `is`) to False. It reduces the number of squiggles in my IDE. 😅

I've used it in the past for, e.g., [dashboard stats queries](https://github.com/ThePalaceProject/circulation/blob/8cfd6599b0cc6f0b9e24928958ee7890f2806ee9/src/palace/manager/api/admin/dashboard_stats.py#L35-L44).

Reply:
Cool! TIL 😄

Reply:
Nice to return the favor every once in a blue moon!